### ISSUE TITLE
<Tab> component accessibility issue that cannot be manually addressed. 

### ISSUE THREAD

**Issue by Pasoon** [carbon-design-system/ibm-products#1837]
When running the accessibility checker, we run into an issue involving `<Tab>` component in which the layer the issue is occuring cannot be accessed. 
<img width="441" alt="image" src="https://user-images.githubusercontent.com/17908476/160073967-eb6a57fa-5c1f-4271-a4d6-ed5fdc977171.png">
<img width="678" alt="image" src="https://user-images.githubusercontent.com/17908476/160074257-01cd98bb-fb6d-4f2a-8565-4029d717e3c8.png">
<img width="743" alt="image" src="https://user-images.githubusercontent.com/17908476/160074315-fa131203-afff-4dbb-8268-4e2e36d60743.png">
The `<div>` with `role="tabpanel"` cannot be manipulated to add prop title or label to fix this issue.

**Comment by lee-chase** [comment]
@Pasoon is this an issue with Carbon Tabs or are you using one of the components published by this package? Where are you importing the Tab component from?

**Comment by Pasoon** [comment]
@lee-chase we are importing it like so `import {
  Row,
  Column,
  Tabs,
  Tab,
  Card,
  InlineNotification,
  /* future consideration: , ColumnWithBorder, RowWithBorderContainer */
} from '@carbon/ibm-security';`
